python/ray/data/preprocessors/scaler.py

```diff
@@ -9,23 +9,68 @@
 
 
 class StandardScaler(Preprocessor):
-    """Scale values within columns based on mean and standard deviation.
+    r"""Scale and translate each column by its mean and standard deviation, respectively.
 
-    For each column, each value will be transformed to ``(value-mean)/std``,
-    where ``mean`` and ``std`` are calculated from the fitted dataset.
+    The general formula is given by
+
+    .. math::
+
+        x' = \frac{x - \bar{x}}{s}
+
+    where :math:`x` is the column, :math:`x'` is the transformed column,
+    :math:`\bar{x}` is the column average, and :math:`s` is the column's sample
+    standard deviation. If :math:`s = 0` (i.e., the column is constant-valued),
+    then the transformed column will contain zeros.
+
+    .. note::
+        Many models assume that data is distributed like a standard normal distribution.
+        To make your data look like a standard normal distribution, use :py:class:`StandardScaler`.
```

Comment:
Don't really like the way this sounds. Open to suggestions on how to re-word.

Reply:
My understanding is that the expected usage of this preprocessor is _if_ the input data is distributed like a standard normal distribution. Arbitrary datasets will not become gaussian simply by applying this.

Reply:
Yeah, you're right. Fixed